src/app/components/Form/index.tsx

```diff
@@ -65,7 +65,7 @@ const NumberInput = styled.input`
 
 const CurrencyUnit = styled.span``;
 
-const inputRegex = RegExp(`^\\d+(?:\\\\[.])?\\d*$`);
+export const inputRegex = RegExp(`^\\d+(?:\\\\[.])?\\d*$`);
```

Comment:
Please give a brief comment to explain it in code, thanks

Reply:
@shallcdn it was an old copy paste code from earlier. Refactored now, thanks for pointing that out!